source/docs/v3/integrations/sail.blade.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Laravel Sail integration
+extends: _layouts.documentation
+section: content
+---
+
+# Laravel Sail {#sail}
+
+## Creating tenants
+
+By default sail has the `ALL` privilege on the default database (here it is the central database).
+In order to create, read, update and delete operations related to the database, we will assign `ALL` privilege on all databases via the wildcard `*`;
+
+To do so either login to the mysql shell or via any client using root user credentials and run the following statements
```

Comment:
@lukinovec can you rewrite this to improve the grammar a bit. You don't need to keep all of the paragraphs, feel free to make it as short as it needs to be.

Reply:
```suggestion
Sail can only perform the create, read, update and delete operations in the central database by default. To give Sail permission also to perform these operations in the tenant databases, log in to Sail's MySQL shell as the root user (by default, the username is `root` and the password is `password`) and assign the `ALL` privilege to all the databases by running the following statements:
```

The original text also didn't provide clear info, I think this suggestion should correct that. Originally, in the first paragraph, it's said that Sail has the privilege in the default DB, but in the next, "in order to do the operations related to the database", we assign the privilege to all databases, which is confusing.

Reply:
@lukinovec I edited your suggestion, let me know your thoughts. And yeah, including that command would be useful since that's what most people will use.

Reply:
The edited suggestion looks good. I edited the parentheses after `log in to Sail's MySQL shell as the root user` to include the commands. We can commit the suggestion if you agree with the change @stancl